applications/GeoMechanicsApplication/tests/cpp_tests/test_head_extrapolation_flow_workflow.cpp

```diff
@@ -10,112 +10,81 @@
 //  Main authors:    Jonathan Nuttall
 //
 
-#include <string>
 #include <iostream>
+#include <string>
 
 // Project includes
 #include "custom_workflows/dgeoflow.h"
 #include "flow_stubs.h"
 #include "geo_mechanics_fast_suite.h"
 #include "test_utilities.h"
 
-namespace Kratos::Testing
+namespace
 {
 
-KRATOS_TEST_CASE_IN_SUITE(CalculateExtrapolatedHeadFlow_1, KratosGeoMechanicsIntegrationSuite)
-{
-    auto workingDirectory = "./applications/GeoMechanicsApplication/tests/test_head_extrapolation_custom_workflow_flow";
-    auto projectFile = "ProjectParameters_1.json";
+using namespace Kratos;
+using namespace Kratos::Testing;
 
-    auto execute = Kratos::KratosExecute();
+const auto workingDirectory = std::filesystem::path{"."} / "applications" / "GeoMechanicsApplication" /
+                              "tests" / "test_head_extrapolation_custom_workflow_flow";
 
-    const Kratos::KratosExecute::CriticalHeadInfo critical_head_info(0, 0, 0);
-    const Kratos::KratosExecute::CallBackFunctions call_back_functions(&flow_stubs::emptyLog,
-                                                                 &flow_stubs::emptyProgress,
-                                                                 &flow_stubs::emptyLog,
-                                                                 &flow_stubs::emptyCancel);
-
-    const int status = execute.ExecuteFlowAnalysis(workingDirectory, projectFile, critical_head_info, "", call_back_functions);
+int RunTestCase(int test_case_number)
+{
+    const auto projectFile = std::string{"ProjectParameters_"} + std::to_string(test_case_number) + ".json";
 
-    KRATOS_EXPECT_EQ(status, 0);
+    auto execute = KratosExecute();
 
-    // output_files
-    std::string original = (std::string) workingDirectory + "/test_head_extrapolate_1.orig.res";
-    std::string result = (std::string) workingDirectory + "/test_head_extrapolate_1.post.res";
+    const KratosExecute::CriticalHeadInfo  critical_head_info(0, 0, 0);
+    const KratosExecute::CallBackFunctions call_back_functions(
+        &flow_stubs::emptyLog, &flow_stubs::emptyProgress, &flow_stubs::emptyLog, &flow_stubs::emptyCancel);
 
-    KRATOS_EXPECT_TRUE(TestUtilities::CompareFiles(original, result))
+    return execute.ExecuteFlowAnalysis(workingDirectory.generic_string(), projectFile,
```

Comment:
Tests work when return KratosExecute().ExecuteFlowAnalysis(...). What is your opinion on having line 35?

Reply:
Nice suggestion, indeed makes it shorter and clearer in my opinion